docs/source/features.rst

```diff
@@ -54,6 +54,31 @@ To use ``docker``:
 When used this way, `tartufo` will clone the repository to a temporary
 directory, scan the local clone, and then delete it.
 
+Scanning a Folder
++++++++++++++++++++++++++++
+
+This is scanning a local folder and does not look at git commit history. Ideal for locating
+secrets in the latest source files or files not in source control.
```

Comment:
These sentences read a little bit awkwardly. Could we update them to something a little more friendly, maybe along the lines of

> Operating in this mode, tartufo scans the files in a local folder, rather than operating on git commit history. This is ideal for locating secrets in the latest version of source files, or files not in source control.

Reply:
Sure. I was matching the wording of the other mode.